exasol_sphinx_github_pages_generator/generate_index.py

```diff
@@ -67,8 +68,10 @@ def get_releases(target_branch: str, target_branch_exists_remote: bool, source_b
     file. Additionally, also return the current release and index.html-file-path. The current release is titled "latest".
     Found releases are returned as a list of dictionaries.
     In order to find the releases in the target_branch, it is checked out in an additional temporary worktree.
-    Takes the root-directory name for each release and returns it. Gives bad results if directory names contain slashes.
-    Please remove slashes from directory names before using them to generate the root directory for your
+    Takes the root-directory name for each release and returns it.
+    Gives bad results if branch names contain slashes, because these result in nested directories instead of
+    one directory, therefore only the first part of the branch name is found by this function.
+    Please remove slashes from branch names before using them to generate the root directory for your
```

Comment:
we already repalce everywhere slashed from branch names, could we do here the same?

Reply:
No, because this reads the paths of files already in the github pages. if these are in nested dirs for some reason, this will give bad results, because there is no way of knowing how many of these dirs are part of the original branch name. i added it to the comment so future users/devs will know this. if all documentation is generated with this tool, this will not be an issue.